docs/content/modeling/getting-started.mdx

```diff
@@ -83,17 +72,13 @@ When you are modeling, you need to answer a more general question:
   />
 </div>
 
-If you can answer that question for all types of objects in your system, then you can codify that into an authorization model.
-
-Let's get started!
+Answering that question for all object types in your system codifies your authorization model.
```

Comment:
I don't think this is saying the same thing as the previous sentence.

It used to say, finish these steps then you can write your authorization model, now it says: finishes these steps and you would have written your authorization model

Reply:
How about:

Answering that question for all object types in your system means your authorization model can be codified.